### PR TITLE
markdownify docs

### PR DESCRIPTION
**Pull request by giladgray**
Sorry this is frighteningly large, but it just has to happen.

TODO @llorca:
- [x] remove `---` front matter from components files
- [x] re-indent code blocks
- [x] remove stray `Weight`s 
- [x] remove extra whitespace
- [x] remove `/* docs */` from scss files, but keep markup + modifiers
- [x] convert datetime package
- [x] convert table package

### CODE REVIEW

**Comment by themadcreator** [comment]
Woo! it's happening!

**Comment by giladgray** [comment]
the build on this feature branch will be broken until documentalist is both open source and published to NPM. then I will open a series of new PRs that replace KSS with documentalist and _voila!_

File: packages/table/src/docs.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
+---
```
Comment (llorca) [inline packages/table/src/docs.md]: Are we putting the copyright header in the .md files? I omitted it from `core` and `datetime`

Comment (adidahiya) [inline packages/table/src/docs.md]: nah, let's not, feels like overkill.

**Pull request merged**